pkg/webhook/add_label_on_review_approval.go

```diff
@@ -17,42 +17,35 @@ package webhook
 import (
 	"context"
 	"fmt"
-	"net/http"
 	"strings"
 
 	"github.com/google/go-github/github"
 	"github.com/pkg/errors"
 	"github.com/syndesisio/pure-bot/pkg/config"
+	"go.uber.org/zap"
 )
 
-var addLabelOnReviewApprovalHandler Handler = &addLabelOnReviewApproval{}
-
 const (
-	approvedLabel       = "approved"
-	approvedReviewState = "approved"
+	approvedLabel  = "approved"
+	assignedAction = "approved"
```

Comment:
This looks a bit fuzzy. See bellow.

Reply:
good point, as I havent touched that functionality at all.

I'm blaming IDEA refactoring for now (much better than blaming myself ;-)

going to revert this ...